ldevnet/ldevnet.go

```diff
@@ -1,374 +0,0 @@
-package ldevnet
```

Comment:
Most of here is now in https://github.com/textileio/lotus-devnet
This changed since this worked on `master`, so I had to migrate to `testnet/3`.